README.md

```diff
@@ -39,13 +39,14 @@ not the actual file-system path).
 Set how "dotfiles" are treated when encountered. A dotfile is a file
 or directory that begins with a dot ("."). Note this check is done on
 the path itself without checking if the path actually exists on the
-disk. If `root` is specified, only the dotfiles above the root are
+disk. If `root` is specified, only dotfiles below the root are
 checked (i.e. the root itself can be within a dotfile when when set
 to "deny").
 
-  - `'allow'` No special treatment for dotfiles.
-  - `'deny'` Send a 403 for any request for a dotfile.
-  - `'ignore'` Pretend like the dotfile does not exist and 404.
+  - `undefined` dotdirs have no special treadment, dotfiles are 404'd.
```

Comment:
Please, absolutely remove `undefined` as an option; this is confusing and will confuse a lot of people. People do not provide an option with the value of `undefined` under normalize circumstances; the code in the module looks for that value, yes, because it's what it means to not have supplied a value in JavaScript.